src/utils/teams.ts

```diff
@@ -0,0 +1,118 @@
+import { DomainColor } from '~gql';
+
+export const setTeamColor = (color?: DomainColor) => {
```

Comment:
I know you've only moved this to a different file but could you also update the naming? These functions aren't setting anything